CollAction/Controllers/ProjectsController.cs

```diff
@@ -464,21 +459,25 @@ public ViewResult StartInfo()
         }
 
         [HttpGet]
-        public async Task<JsonResult> GetTileProjects(int? categoryId, int? statusId)
+        public async Task<JsonResult> GetTileProjects(int? projectId)
```

Comment:
This API is used for other things. Normal project search is by category and status... so I'm pretty sure this breaks things. We should probably get a different API endpoint for this.

Reply:
Restored original function and added a new one to get just one project.